src/Client.js

```diff
@@ -77,6 +79,7 @@ function Client(options) {
   this._observer = opts.observer
   this._lastSeen = null
   this._headers = opts.headers
+  this._fetch = opts.fetch || require('cross-fetch')
```

Comment:
is it common to use `require(...)` in the middle of the code?

Reply:
It depends. It is used to avoid loading unnecessary stuff (on front-end side it reduces the bundle size).